vendor/inline-scripts/mediawiki-scripts-handlers.js

```diff
@@ -0,0 +1,53 @@
+(function () {
+	var gettersQueue = [];
+	var callbacksQueue = [];
+	var isListening = false;
+
+	window.onAsyncScriptsLoaded = function () {
```

Comment:
How about using `window.M` instead of polluting the global namespace?

Reply:
DO you think it's safe to set properties on window.M outside Mercury-shared?

Reply:
Huh, I thought we already do this but we don't. This could be a bad practice, I'm taking it back :slightly_smiling_face: